lms/djangoapps/courseware/management/commands/dump_course_structure.py

```diff
@@ -165,30 +165,37 @@ def dump_module_by_position(course_id, module, level=0,
     else:
         display_name_long = prefix + "," + module.display_name
 
-    if destination == None:
-        destination = [ ('course_id',
-                         'position',
-                         'level',
-                         'module_id',
-                         'type',
-                         'displayname',
-                         'path',
-                         'parent') ]
-
-    destination.append( (course_id,
-                         pos, 
-                         level, 
-                         module.id, 
-                         module.location.category,
-                         module.display_name, 
-                         display_name_long,
-                         parent) )
+    if destination is None:
+        destination = [(
+            'course_id',
+            'position',
+            'level',
+            'module_id',
+            'type',
+            'displayname',
+            'path',
+            'parent')]
+
+    #  This was a bit confusing for me - was this the intent?
+    #  I think 'extend' is for appending a sequence...
+    #  ...which means we need to use parens to close the tuple?
+    destination.append((
```

Comment:
You may even want to drop this down a line and reindent for increased legibility:

```python
destination.append(
    (
        course_id,
        '...',
    )
)
```

This makes it clear that you're appending a tuple to a list.